clap-v3-utils/Cargo.toml

```
@@ -15,6 +15,10 @@ targets = ["x86_64-unknown-linux-gnu"]
 [lib]
 name = "solana_clap_v3_utils"
 
+[features]
+default = ["elgamal"]
+elgamal = ["solana-zk-sdk"]
```

nit: can we use the explicit `dep` syntax for this?
```suggestion
elgamal = ["dep:solana-zk-sdk"]
```